recipes/mold/all/conanfile.py

```diff
@@ -0,0 +1,86 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.45.0"
+
+class MoldConan(ConanFile):
+    name = "mold"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/rui314/mold/"
+    license = "Zlib"
+    description = ("mold is a faster drop-in replacement for existing Unix linkers. It is several times faster than the LLVM lld linker")
+    topics = ("mold", "ld", "linkage", "compilation")
+
+    settings = "os", "arch", "compiler", "build_type"
+
+    generators = "make"
+
+    def validate(self):
+        if self.settings.os == "Windows":
+            raise ConanInvalidConfiguration('{} can\'t be built on {}.'.format(self.name, self.settings.os))
+        if self.settings.compiler == "gcc" and tools.Version(self.settings.compiler.version) <= "10":
+            raise ConanInvalidConfiguration("GCC version 10 or higher required")
+        if (self.settings.compiler == "clang" or self.settings.compiler == "apple-clang") and tools.Version(self.settings.compiler.version) <= "11":
+            raise ConanInvalidConfiguration("Clang version 11 or higher required")
+        if self.settings.compiler == "apple-clang" and "arm" in self.settings.arch :
+            raise ConanInvalidConfiguration('Use apple-clang does not work on ARM with this recipe')
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def _get_include_path(self, dependency):
+        include_path = self.deps_cpp_info[dependency].rootpath
+        include_path = os.path.join(include_path, "include")
+        return include_path
+
+    def _patch_sources(self):
+        tools.replace_in_file("source_subfolder/Makefile", "-Ithird-party/xxhash ", "-I{} -I{} -I{}".format(
+        self._get_include_path("zlib"),
+        self._get_include_path("openssl"),
+        self._get_include_path("xxhash")))
+
+        tools.replace_in_file("source_subfolder/Makefile", "-Ithird-party/mimalloc/include", "-I{}".format(
+        self._get_include_path("mimalloc")))
+
+        tools.replace_in_file("source_subfolder/Makefile", "-Ithird-party/tbb/include", "-I{}".format(
+        self._get_include_path("onetbb")))
+
```

Comment:
```suggestion

def package_id(self):
   del self.info.settings.compiler

```
https://github.com/conan-io/conan-center-index/blob/master/docs/packaging_policy.md#settings